core/src/test/resources/sql/sub-query.iq

```diff
@@ -2814,7 +2814,7 @@ EnumerableCalc(expr#0..13=[{inputs}], expr#14=[<>($t10, $t9)], expr#15=[1], expr
         EnumerableAggregate(group=[{7}])
           EnumerableTableScan(table=[[scott, EMP]])
         EnumerableCalc(expr#0..5=[{inputs}], expr#6=[CAST($t1):BIGINT NOT NULL], expr#7=[CAST($t2):BIGINT NOT NULL], expr#8=[CAST($t5):BOOLEAN NOT NULL], DEPTNO=[$t0], c=[$t6], d=[$t7], dd=[$t3], m=[$t4], trueLiteral=[$t8])
-          EnumerableAggregate(group=[{1}], c=[MIN($2) FILTER $7], d=[MIN($3) FILTER $7], dd=[COUNT($0) FILTER $6], m=[MIN($4) FILTER $7], trueLiteral=[MIN(true, $5) FILTER $7])
+          EnumerableAggregate(group=[{1}], c_g0=[MIN($2) FILTER $7], d_g0=[MIN($3) FILTER $7], dd_g0=[COUNT($0) FILTER $6], m_g0=[MIN($4) FILTER $7], trueLiteral_g0=[MIN(true, $5) FILTER $7])
```

Comment:
Here is the case of MIN with FILTER. There are many more in agg.iq.

Reply:
ok, great, thanks